### PR TITLE
v3.1

### PR DESCRIPTION
**Pull request by rafatosta**
<release date="2022-07-21" version="3.1">
      <description>
          <ul>
              <li>Spellchecker</li>
              <li>Add Italian translation</li>
              <li>Add Dutch translation</li>
          </ul>
        </description>
    </release>

### CODE REVIEW

File: com.rtosta.zapzap.yaml [hunk 1]
```diff
@@ -25,9 +25,10 @@ cleanup:
   - /lib/pkgconfig
   - '*.la'
   - '*.a'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
 modules:       
-  - pyqt6.yml
-  - pyqt6-webengine.yml
+  - name: PyQtWebEngineApp
```
Comment (hfiguiere) [inline com.rtosta.zapzap.yaml:31]:
```suggestion
```

This an empty module.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.rtosta.zapzap.yaml (hunk 1 above)
Comment (rafatosta) [reply com.rtosta.zapzap.yaml:31]: I didn't understand the change. 
I followed the documentation: https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp/wiki

Comment (hfiguiere) [reply com.rtosta.zapzap.yaml]: And the documentation use this to indicate "your app". Not an empty module like you did. There is nothing special.

Comment (hfiguiere) [reply com.rtosta.zapzap.yaml]: So my change as suggested is to remove the line.

Comment (rafatosta) [reply com.rtosta.zapzap.yaml]: I noticed it later when it flagged. Thanks

Comment (rafatosta) [reply com.rtosta.zapzap.yaml]: I don't understand why it failed after the merge

Comment (hfiguiere) [reply com.rtosta.zapzap.yaml]: infra problem, I did retrigger.

**Pull request merged**